### PR TITLE
Formatted with Black

### PR DESCRIPTION
**Pull request by phicks22**
@gwaybio I forgot to format my new changes for the last PR.

### CODE REVIEW

File: IDR/production/1.compute_statistics.py [hunk 1]
```diff
@@ -79,10 +88,10 @@ def nested_dict_pairs_iterator(elements_and_counts_dict):
...
         columns=["Study", "Screen", "Attribute", "Element", "Count"],
     )
-    
-    unique_elements_and_counts_df["Element"] = unique_elements_and_counts_df["Element"].apply(
-            lambda x: str(x)
-        )
+
+    unique_elements_and_counts_df["Element"] = unique_elements_and_counts_df[
+        "Element"
+    ].apply(lambda x: str(x))
```
Comment (d33bs) [inline IDR/production/1.compute_statistics.py]: Consider using `astype` here to increase readability.

```suggestion
    ].astype("str")
```

**Review by d33bs — COMMENTED** [review]
Nice job! I left a minor suggestion for your consideration loosely based on the PR title/focus (code style/readability). 

I also noticed that there may be other files which might need `black` formatting (discovered via the command: `black ./ --check`). Would these benefit from being changed along with the others in this PR?

**Comment by phicks22** [comment]
> Nice job! I left a minor suggestion for your consideration loosely based on the PR title/focus (code style/readability).
> 
> I also noticed that there may be other files which might need `black` formatting (discovered via the command: `black ./ --check`). Would these benefit from being changed along with the others in this PR?

@d33bs Which files?? I ran the `black ./ --check` for the repo and it said  21 files would be left unchanged. Thanks for taking a look at this.

File: IDR/production/1.compute_statistics.py (hunk 1 above)
Comment (phicks22) [reply IDR/production/1.compute_statistics.py]: @d33bs Ah I didn't know you could do that for pandas. I changed it to `astype("str")`.

Added in e638f2c.

**Comment by d33bs** [comment]
@phicks22 - apologies, I reran `--check` and found the same, no additional files need formatting. I was in the another environment which had very minor formatting differences, please disregard my earlier comment.

**Comment by phicks22** [comment]
> @phicks22 - apologies, I reran `--check` and found the same, no additional files need formatting. I was in the another environment which had very minor formatting differences, please disregard my earlier comment.

@d33bs Perfect, thank you!

**Pull request merged**